lib/eex/lib/eex/compiler.ex

```diff
@@ -58,7 +58,22 @@ defmodule EEx.Compiler do
     {marker, t} = retrieve_marker(t)
 
     case expr(t, line, column + 2 + length(marker), state, []) do
-      {:error, line, column, message} ->
+      {:error, _line, _column, message} ->
```

Comment:
@josevalim I'm changing here to use the line and column from the expression that we couldn't find the closing `%>`. So we can display the error in the line we have the error:

![image](https://user-images.githubusercontent.com/27698968/198378855-4e2a441a-6940-435b-b3bd-457323455bb7.png)

Otherwise it would should only in the end of the line. Please, let me know if I'm missing something.